modules/plugin/jdbc/jdbc-hana/pom.xml

```diff
@@ -17,7 +17,6 @@
     <groupId>org.geotools.jdbc</groupId>
     <artifactId>gt-jdbc</artifactId>
     <version>22-SNAPSHOT</version>
-    <relativePath>../../../plugin/jdbc/pom.xml</relativePath>
```

Comment:
This looks wrong?

Reply:
It is the same as in the other jdbc-plugins. The only have elements groupId, artifactId and version.

Reply:
Sorry I misread the previous, so what was odd was previously this module declared the "wrong" parent :P